### ISSUE TITLE
DEPLOY - Multiple errors, meteor build on windows all-in-one project

### ISSUE THREAD

**Issue by bouchepat** [Urigo/angular-meteor#1852]
I've been trying to create a build on windows for a deployment on linux. Since I was not able to get it to work with my app I used the all-in-one project included in this repo. This is what I run:
```
meteor build ../packaged --architecture os.linux.x86_64
```
This is what I get:
```
C:\Users\Patrick\Meteor\all-in-one>meteor build ../output --architecture os.linux.x86_64
[client]: ES2015 modules Compilation: 562.721ms
AoT Code Generator Created.: 1347.817ms
NgModules Loaded.: 0.115ms
[client]: HTML Files Compilation: 6.760ms    -
[client]: SCSS Files Compilation: 0.855ms    -
[server]: ES2015 modules Compilation: 265.730ms
AoT Code Generator Created.: 1106.121ms
NgModules Loaded.: 0.076ms
[server]: HTML Files Compilation: 1.381ms    |
[server]: SCSS Files Compilation: 0.623ms    |
Errors prevented bundling:
While processing files with angular-compilers (for target web.browser):
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-scss-compiler\node_modules\node-sass\lib\index.js:439:16: Z
at Object.module.exports.renderSync
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-scss-compiler\node_modules\node-sass\lib\index.js:439:16)
at Function.compileFile (packages/angular-scss-compiler/index.js:19:17)
at TsCompilerAotCompilerTypeCheckHostAdapter.compiler._host.loadResource.filePath (packages/angular-typescript-compiler/index.js:317:38)
at Object.get
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler_factory.js:78:306)
at DirectiveNormalizer._fetch
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:95:7)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:292:36
at Array.map (<anonymous>)
at DirectiveNormalizer._loadMissingExternalStylesheets
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:291:14)
at DirectiveNormalizer._normalizeTemplateMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:201:7)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:127:7
at Object.then
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\util.js:164:3)
at DirectiveNormalizer.normalizeTemplate
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:127:7)
at CompileMetadataResolver.loadDirectiveMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:361:44)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:614:54
at Array.forEach (<anonymous>)
at CompileMetadataResolver.loadNgModuleDirectiveAndPipeMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:613:16)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler.js:91:7
at Array.map (<anonymous>)
at AotCompiler.analyzeModulesAsync
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler.js:91:7)
at Promise.asyncApply (packages/angular-typescript-compiler/index.js:332:42)
at C:\Users\Patrick\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\meteor-promise\fiber_pool.js:43:40


While processing files with angular-compilers (for target os.linux.x86_64):
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-scss-compiler\node_modules\node-sass\lib\index.js:439:16: p
at Object.module.exports.renderSync
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-scss-compiler\node_modules\node-sass\lib\index.js:439:16)
at Function.compileFile (packages/angular-scss-compiler/index.js:19:17)
at TsCompilerAotCompilerTypeCheckHostAdapter.compiler._host.loadResource.filePath (packages/angular-typescript-compiler/index.js:317:38)
at Object.get
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler_factory.js:78:306)
at DirectiveNormalizer._fetch
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:95:7)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:292:36
at Array.map (<anonymous>)
at DirectiveNormalizer._loadMissingExternalStylesheets
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:291:14)
at DirectiveNormalizer._normalizeTemplateMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:201:7)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:127:7
at Object.then
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\util.js:164:3)
at DirectiveNormalizer.normalizeTemplate
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\directive_normalizer.js:127:7)
at CompileMetadataResolver.loadDirectiveMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:361:44)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:614:54
at Array.forEach (<anonymous>)
at CompileMetadataResolver.loadNgModuleDirectiveAndPipeMetadata
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\metadata_resolver.js:613:16)
at
C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler.js:91:7
at Array.map (<anonymous>)
at AotCompiler.analyzeModulesAsync
(C:\Users\Patrick\AppData\Local\.meteor\packages\angular-compilers\0.2.9_2\plugin.Angular_Compilers.os\npm\node_modules\meteor\angular-typescript-compiler\node_modules\packages\compiler\esm5\src\aot\compiler.js:91:7)
at Promise.asyncApply (packages/angular-typescript-compiler/index.js:332:42)
at C:\Users\Patrick\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\meteor-promise\fiber_pool.js:43:40
```
However, even with the error a bundle is created in: 

all-in-one/.meteor/local/build_tar/bundle

but not where it is supposed to.

When I delete the build_tar folder and re-run the same command, NO error are generated but the bundle_tar is no longer created. I need to do a meteor reset and rere-run the command (which trigger the error again).

Any suggestions would be appreciated.

**Comment by bouchepat** [comment]
So even with all the errors mentioned in previous comment, I managed to get a tarball for my app, I deployed it on a linux environment with nginx/passenger. Unfortunately I'm getting 
```
Cannot find module '@angular/core'
```
in the dev tool chrome. I'm guessing the errors generated during the build are the cause of this.

**Comment by bouchepat** [comment]
Anybody able to deploy any of the examples, tried manually and with mup but no luck.

**Comment by ardatan** [comment]
Closed by #1889
